scss/components/_top-bar.scss

```diff
@@ -155,8 +155,8 @@ $topbar-unstack-breakpoint: medium !default;
   @else {
     .top-bar-title {
       float: left;
-      margin-right: $topbar-title-spacing;
-
+      margin: $topbar-title-spacing;
+      display:inline-block;
```

Comment:
I happened to glance at that exact same article right before asking the above question. Thanks for the info.